discovery-provider/plugins/pedalboard/README.md

```diff
@@ -0,0 +1,25 @@
+# Pedalboard 🎸
+
+Pedalboard is a plugin framework for Discovery nodes! It aims to provide the developer with a set of tools and utilities so they can spend less time worrying about setup and scalability and more about cool features. 
+
+# Tools
+
+Pedalboard uses a number of tools to make life easy.
+
+typescript: typed js
+
+knex-types: a typescript codegen tool to make working with db entities painless
+
+knex: query builder for ts and js
+
+…
+
+more as pedalboard gets built out
+
+# Running against containerized DBs
```

Comment:
the ssh tunnel is needed for containerized DBs? or for remote DBs?

Reply:
ah, remote dbs